src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -315,6 +368,14 @@ class SystemInformation(BaseModel):
 
 
 class TracklogEvent(BaseModel):
+    """The tracklog block contains a record of events recorded on these data. This is experimental for now.
```

Comment:
Suggest remove the part about it being experimental - it is used and depended on.